### PR TITLE
gNOI-3.4: ensure that gnoi reboot status request is sent for the active rp when the device is dual-rp

### PR DESCRIPTION
**Pull request by mojiiba**
Our current implementation saves the context of the gnoi reboot with delay for the active rp. Thus, the component name for status request should be the active rp.

### CODE REVIEW

**Review by prinikasn — COMMENTED** [review]
/gcbrun

File: feature/gnoi/system/tests/chassis_reboot_status_and_cancel_test/chassis_reboot_status_and_cancel_test.go [hunk 1]
```diff
@@ -206,3 +188,16 @@ func TestCancelReboot(t *testing.T) {
...
 }
+
+func getSubCompPath(t *testing.T, dut *ondatra.DUTDevice) *tpb.Path {
+	t.Helper()
+	controllerCards := components.FindComponentsByType(t, dut, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CONTROLLER_CARD)
+	if len(controllerCards) == 0 {
+		t.Fatal("No controller card components found in DUT.")
+	}
+	activeRP := controllerCards[0]
+	if len(controllerCards) == 2 {
```
Comment (prinikasn) [inline feature/gnoi/system/tests/chassis_reboot_status_and_cancel_test/chassis_reboot_status_and_cancel_test.go:199]: Would there be a possible scenario where controllerCards is greater than 2?

Comment (mojiiba) [reply feature/gnoi/system/tests/chassis_reboot_status_and_cancel_test/chassis_reboot_status_and_cancel_test.go:199]: I am not aware of a device with more than two rps, so the length should be 1 or two.

Comment (vosipchu) [reply feature/gnoi/system/tests/chassis_reboot_status_and_cancel_test/chassis_reboot_status_and_cancel_test.go:199]: +1, we will have devices with 2 control modules only

**Review by prinikasn — APPROVED** [review]

**Pull request merged**